lib/arelastic/aggregations/aggregation.rb

```diff
@@ -16,6 +16,10 @@ def nested(path)
         Arelastic::Aggregations::Nested.new(name, path, [self])
       end
 
+      def reverse_nested(path = nil)
+        Arelastic::Aggregations::ReverseNested.new(name, path, [self])
+      end
```

Comment:
Yes.  Reverse Nested aggregations let you jump from one nested path to another.

In practice, this is hard to build and parse versus the alternative:  reverse nest to root, and then immediately nest to another path.  However, I support it because ES supports it.